roles/usegalaxy-eu.remove-orphan-condor-jobs/files/remove-orphan-condor-jobs

```diff
@@ -11,7 +11,7 @@ condor_q -autoformat Cmd ClusterId JobStatus | grep " 2$" | cut -d " " -f1,2 | a
 while IFS=' ' read col1 col2; do
 
     # Remove the Job from the condor cluster if the Galaxy ID was not found in galaxy_running_jobs
-    if ! [[ " $col1 " == " ${galaxy_running_jobs[@]} " ]]; then
+    if [[ ! " ${galaxy_running_jobs[*]} " =~ " ${col1} " ]]; then
```

Comment:
Do you want maybe to log this and/or also add a --dry-run option?

Reply:
https://github.com/usegalaxy-eu/infrastructure-playbook/pull/1040